tools/chapel-py/src/method-tables.h

```diff
@@ -611,6 +613,11 @@ CLASS_BEGIN(StringParam)
                UniqueString, return node->value())
 CLASS_END(StringParam)
 
+CLASS_BEGIN(CompositeType)
+  PLAIN_GETTER(CompositeType, decl, "Get the location where this composite type is declared",
```

Comment:
this docstring seems to imply something else. Maybe something like `"Get the AstNode which declares this composite type"`?